apache/datadog_checks/apache/apache.py

```diff
@@ -68,16 +72,19 @@ def check(self, instance):
             raise
         else:
             self.service_check(service_check_name, AgentCheck.OK, tags=service_check_tags)
-        server = r.headers.get("Server")
-        if server:
-            self._collect_metadata(server)
+
         self.log.debug("apache check succeeded")
         metric_count = 0
+        version_submitted = False
         # Loop through and extract the numerical values
         for line in r.iter_lines(decode_unicode=True):
             values = line.split(': ')
             if len(values) == 2:  # match
                 metric, value = values
+                # Special case: ServerVersion
+                if metric == 'ServerVersion':
+                    self._submit_metadata(value)
+                    version_submitted = True
```

Comment:
There should be a `continue` here no?

Reply:
It's a no-op as the rest of the loop doesn't handle that content.
But for consistency yes it's better, I'll add it